tools/serverpod_cli/bin/generator/class_generator_dart.dart

```diff
@@ -32,6 +32,7 @@ class ClassGeneratorDart extends ClassGenerator {
     var out = '';
 
     String? tableName = classDefinition.tableName;
+    bool? isViewTable = classDefinition.viewTable;
```

Comment:
Should not be nullable. E.g. `var isViewTable = classDefinition.isViewTable ?? false`